packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1140,7 +1140,20 @@ export const MQA_INTEVAL = 60000; // mqa analyzer interval its fixed to 60000
 // Metrics constants ----------------------------------------------------------
 
 export const METRICS_OPERATIONAL_MEASURES = {
+  ADD_MEDIA_FAILURE: 'js_sdk_add_media_failure',
```

Comment:
The measure name on existing is "failures" e.g. `js_sdk_get_user_media_failures`, not sure if you want to keep consistent and make the new measure names plural as well.